src/Widgets/ButtonWidget/ButtonWidgetEditingConfig.js

```diff
@@ -32,4 +32,27 @@ Scrivito.provideEditingConfig("ButtonWidget", {
     alignment: "left",
     style: "btn-primary",
   },
+  validations: [
+    [
+      "target",
+
+      target => {
+        if (!target) {
+          return {
+            message: "The link target should be set.",
+            severity: "warning",
+          };
+        }
+      },
+    ],
+    [
+      "title",
```

Comment:
The ButtonWidget does not have a distinct `title` attribute, but used the title of the target.

So this can be merged into the `"target"` validation as `!target.title()`.